internal/hcsoci/layers.go

```diff
@@ -141,7 +141,7 @@ func MountContainerLayers(ctx context.Context, layerFolders []string, guestRoot
 	}
 
 	hostPath := filepath.Join(layerFolders[len(layerFolders)-1], "sandbox.vhdx")
-	containerScratchPathInUVM := ospath.Join(uvm.OS(), guestRoot, scratchPath)
+	containerScratchPathInUVM := ospath.Join(uvm.OS(), guestRoot)
```

Comment:
We should probably rename this now? Or do you think it still makes sense to call it this?

Reply:
I guess its still technically correct

Reply:
I think it still makes sense. I can change it if you think it would be better though.

Reply:
Sounds fine the more I think about it.